lib/api/controllers/adminController.js

```diff
@@ -41,6 +46,9 @@ class AdminController extends BaseController {
   constructor(kuzzle) {
     super(kuzzle, [
       'dump',
+      'loadFixtures',
+      'loadMappings',
+      'loadSecurities',
```

Comment:
As discussed, we're stuck with that name for the time being, because it has already been "released" (well, documented at least)

But I agree, loadRights or loadPermissions would be better suited